File: nodejs/7 kyu/156.js
// https://www.codewars.com/kata/59727ff285281a44e3000011/train/javascript

// Passed

function bandNameGenerator(str) {
  let bandName;
  if (str[0] === str[str.length - 1]) {
    bandName = str[0].toUpperCase() + str.slice(1) + str.slice(1);
  } else {
    bandName = "The " + str[0].toUpperCase() + str.slice(1);
  }

  return bandName;
}

const output = bandNameGenerator("sandles");
console.log(output);

File: python/7 kyu/156.py
# https://www.codewars.com/kata/6071ef9cbe6ec400228d9531/train/python

# Passed

def calculator(txt):
    get_first_and_second_term = lambda operator: (len(txt.split(operator)[0].strip()), len(txt.split(operator)[1].strip()))

    if "+" in txt:
        first_term, second_term = get_first_and_second_term("+")
        result = (first_term + second_term) * "."
    elif "-" in txt:
        first_term, second_term = get_first_and_second_term("-")
        result = (first_term - second_term) * "."
    elif "*" in txt:
        first_term, second_term = get_first_and_second_term("*")
        result = (first_term * second_term) * "."
    else:
        first_term, second_term = get_first_and_second_term("//")
        result = (first_term // second_term) * "."
        
    return result
        
output = calculator("..... + ...............")
print(output)